pkg/broker/deprovision_subscriber.go

```diff
@@ -102,12 +77,12 @@ func cleanupDeprovision(instance *apb.ServiceInstance, dao *dao.Dao) error {
 	id := instance.ID.String()
 
 	if err = dao.DeleteExtractedCredentials(id); err != nil {
-		log.Error("failed to delete extracted credentials - %#v", err)
+		log.Error("failed to delete extracted credentials - %v", err)
```

Comment:
any reason you changed from `%#v` to `%v`? @shawn-hurley I think I've seen you use the `%#v` variant, thoughts?

Reply:
There seemed to be different approaches to this across the code base with ```%v``` seeming to be the most common. In my opinion, the %v is good for error messages as it will just print the error string, where as perhaps the %#v is good for debug statements as it prints the struct
```
&errors.errorString{s:"test error"}
```
see example https://play.golang.org/p/nkVaxztAG4_I